Assets/Plugins/ARFoundationTrackingPlugin/Scripts/Hado/ARFoundation/WorldAnchorManager.cs

```diff
@@ -39,6 +39,8 @@ public class WorldAnchorManager : MonoBehaviour
         private void Start()
         {
             var moveStartTransform = gameObject.transform;
+            worldAnchorMatrix = Matrix4x4.TRS(gameObject.transform.position, gameObject.transform.rotation,

```

Comment:
```suggestion
            worldAnchorMatrix = Matrix4x4.identity;
```
デフォルト値だろうしTRS計算しなくても良さそう？

Reply:
world anchor の初期位置が何かしらの理由でズレた場合は合わせた方がいいと思っていて、多分ズレないですが

Reply:
異常なズレだと思うから計算しない方が良さそうに思える、どうだろ

Reply:
マーカー見るまではただのgameObjectなのでズレるってことはないですね

Reply:
初回１回だけなら、計算したほうがベターと思います。原点にいることを前提にしたコードにすると意図せずバグを生みそうです。